editor/package.json

```diff
@@ -12,8 +12,8 @@
   },
   "scripts": {
     "clean": "rm -rf lib/",
-    "check": "tsc --noEmit && npm run lint && npm run check-dependency-cruiser && npm run test",
-    "check-ci": "tsc --noEmit && npm run lint && npm run check-dependency-cruiser && npm run test-ci",
+    "check": "tsc --noEmit && npm run lint && npm run check-dependency-cruiser && npm run prettier-check && npm run test",
```

Comment:
I'm very confident that `check-ci` should run `prettier-check`, I'm less sure about `check` here. Does anyone have a workflow that would be hurt by this inclusion here? maybe @seanparsons  ?

Reply:
`check` should absolutely be running the same checks that `check-ci` does